opflexagent/utils/bridge_managers/ovs_lib.py

```diff
@@ -26,9 +28,11 @@ def __init__(self, *args, **kwargs):
         super(OVSBridge, self).__init__(*args, **kwargs)
 
     def reset_ofversion(self):
-        context = self.ovsdb.context
-        return impl_vsctl.BaseCommand(context, 'set',
-                                      args=[self.br_name, 'protocols=[]'])
+        # REVISIT: impl_vsctl is gone in stein!
```

Comment:
ditto

Reply:
will fix (along with the other one I accidentally resolved).